projects/isy-angular-widgets-demo/src/app/feature/objekt-suchen/objekt-suchen.component.spec.ts

```diff
@@ -571,4 +544,19 @@ describe('Integration Tests: PersonenSuchenComponent', () => {
     enableClearSearch = component.enableClearSearch();
     expect(enableClearSearch).toBeTrue();
   });
+
+  it('should find person', () => {
+    const findPersonSpy = spyOn(spectator.component, 'findPerson');
+    expect(component.tbLoadingStatus).toBeFalse();
+
+    const searchButton = spectator.query('#search-button') as HTMLButtonElement;
+    searchButton.addEventListener('click', function () {
+      spectator.component.findPerson();
+    });
+    searchButton.click();
+    spectator.fixture.detectChanges();
+
+    expect(findPersonSpy).toHaveBeenCalled();
+    expect(component.tbLoadingStatus).toBeFalse();
+  });
```

Comment:
#### ℹ️ Optional Improvement
The idea of the test is good, the implementation could be improved. Instead of testing if the internal method would be called it should be testet if the result list contains a row with persons that match the given search criterial from the form above after clicking the search button.

Reply:
New ticket needed.
Actually no better solution found